src/filesystem.cpp

```diff
@@ -18,6 +18,7 @@
 #ifdef _MSC_VER
 #   include "wdirent.h"
 #   include <direct.h>
+#   define chdir _chdir
```

Comment:
The problem is, we shouldn't be invoking this code except on OSX, we don't need this change on linux or windows.